src/components.js

```diff
@@ -826,14 +827,14 @@ const components = [
     name: 'Fixed',
     type: 'div',
     props: {},
-    style: props => ({
+    style: props => Object.assign({
       position: 'fixed',
       top: props.top ? 0 : null,
       right: props.right ? 0 : null,
       bottom: props.bottom ? 0 : null,
       left: props.left ? 0 : null,
       zIndex: props.z
-    }),
+    }, center(props)),
```

Comment:
I think you could have three separate functions – I'm suggesting pulling the existing logic into its own function for `top`, `right`, `bottom`, and `left` props, keeping your new `center` function` and using another function that wraps those two. I'm mostly thinking about reducing the amount of logic in this `components` array